executor/main.go

```diff
@@ -18,20 +18,23 @@ func Main() {
 	logger.Info("Starting rancher-compose-executor")
 
 	eventHandlers := map[string]events.EventHandler{
-		"environment.create":        handlers.WithLock(handlers.CreateEnvironment),
-		"environment.upgrade":       handlers.WithLock(handlers.UpgradeEnvironment),
-		"environment.finishupgrade": handlers.WithLock(handlers.FinishUpgradeEnvironment),
-		"environment.rollback":      handlers.WithLock(handlers.RollbackEnvironment),
+		"stack.create":        handlers.WithLock(handlers.CreateStack),
+		"stack.upgrade":       handlers.WithLock(handlers.UpgradeStack),
+		"stack.finishupgrade": handlers.WithLock(handlers.FinishUpgradeStack),
+		"stack.rollback":      handlers.WithLock(handlers.RollbackStack),
 		"ping": func(event *events.Event, apiClient *client.RancherClient) error {
 			return nil
 		},
 	}
 
+	logrus.Info(os.Getenv("CATTLE_URL"))
+
 	router, err := events.NewEventRouter("rancher-compose-executor", 2000,
-		os.Getenv("CATTLE_URL"),
+		//os.Getenv("CATTLE_URL"),
```

Comment:
In scripts somewhere.... But it's slightly valid use case.  You should rewrite the url as `/v2-beta`